notebooks/Gemma+Gemini+Eval.ipynb

```diff
@@ -0,0 +1,8838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        gen_input = tokenizer.apply_chat_template(messages, return_tensors="pt", add_generation_prompt=True).to(model.device)
</pre><p> </p><p>Since <code>device_map="auto"</code> we don't need to do the device placement manually :)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/llmops-pipeline/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Didn't know this is applicable to the tokenizer!! Thanks for the comment :)